{"name": "test_add2[【类型】输入空格，给出提示信息1]", "status": "passed", "steps": [{"name": "步骤一： 和3.14开始计算，预期应等于参数大小超出范围，打印<ExceptionInfo for raises contextmanager>", "status": "broken", "statusDetails": {"message": "TypeError: '>' not supported between instances of 'str' and 'int'\n", "trace": "  File \"D:\\zj\\PycharmProjects\\study202208\\study0818\\test\\test_calculate.py\", line 52, in test_add2\n    result=self.cal.add(x,y)\n  File \"D:\\zj\\PycharmProjects\\study202208\\script\\calculator.py\", line 4, in add\n    if a > 99 or a < -99 or b > 99 or b < -99:\n"}, "start": 1660805207772, "stop": 1660805207772}], "attachments": [{"name": "log", "source": "eeff8c94-0560-432b-88ea-2f4473a0a3f0-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "aadc90b0-42f9-4f38-b128-64f91b574e29-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "327200c5-48c3-4024-8b97-c7bd00f067d6-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "x", "value": "' '"}, {"name": "y", "value": "3.14"}, {"name": "expect", "value": "'参数大小超出范围'"}], "start": 1660805207772, "stop": 1660805207773, "uuid": "dadc07f4-2d9f-494e-85db-ed5530025595", "historyId": "e47079f6b402f00730355da220abcb87", "testCaseId": "c1fd4152c4894ab47d6f7448dc1c5bcf", "fullName": "study0818.test.test_calculate.TestAdd#test_add2", "labels": [{"name": "feature", "value": "加法计算模块"}, {"name": "story", "value": "加法报错用例"}, {"name": "tag", "value": "P2"}, {"name": "parentSuite", "value": "study0818.test"}, {"name": "suite", "value": "test_calculate"}, {"name": "subSuite", "value": "TestAdd"}, {"name": "host", "value": "LAPTOP-BEO6NH7O"}, {"name": "thread", "value": "7552-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "study0818.test.test_calculate"}]}